### DISCUSSION TITLE
Is there anyone can update the 'dbproviders.properties'.

### DISCUSSION THREAD

**Discussion by lengluo** [quartznet/quartznet#472] [General] [answered]
I want to use oracle to persistence job info, but I can't find the relative provider because of the version of Oracle.ManagedDataAccess.dll

**Comment by lahma** [comment, marked as answer]
I'm afraid I don't quite understand the question. But you should be able to install just the NuGet package and create needed assembly binding redirect if the version doesn't exactly match. Quartz 2.x is using [this package](https://www.nuget.org/packages/Oracle.ManagedDataAccess/12.1.2400) as base dependency for testing.
